docs/en/internationalization.md

```diff
@@ -0,0 +1,132 @@
+# Contributing to internationalization (i18n)
+
+Thanks to different contributors, FreshRSS is actually translated in 15 languages. This section will explain you the basis of the internationalization in FreshRSS, either if you want to translate the application in your own language or if you have to make a specific change.
+
+## Overview
+
+It is common (and that's an understatement) to want to show some text to the user. The problem is that FreshRSS has users of different nationalities. It is therefore necessary to be able to manage different languages in order not to remain confined to English or French.
+
+The solution is to use the `Minz_Translate` module, which allows dynamic translation of FreshRSS. Before using this module, it is necessary to know where to find the strings to be translated. Each language has its own subdirectory in a parent directory named `app/i18n/`. For example, English language files are located in [`app/i18n/en/`](/app/i18n/en/). There are seven different files:
```

Comment:
Does that relative link work from freshrss.github.io?

Reply:
I hope so: it works in the GitHub interface so I guess it'll work on github.io too